src/main/java/com/google/sps/data/Election.java

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.HashSet;
+
+/** An item on a todo list. */
+public final class Election {
+
+  private final String name;
+  private final String scope;
+  private final HashSet<long> positions;
```

Comment:
Does this work? I thought it need to be HashSet<Long> (autoboxed).

Reply:
Same below.

Reply:
Resolved in later commit.